CMakeLists.txt

```diff
@@ -272,6 +272,9 @@ if(CMAKE_C_COMPILER_ID MATCHES "MSVC")
     /wd5105 # macro expansion producing 'defined' has undefined behavior
   )
 
+  # Terminate building as soon as one error occurs
+  add_compile_options(/errorReport:prompt)
```

Comment:
this is deprecated and relates to ICE (not normal compiler errors about source code) https://docs.microsoft.com/en-us/cpp/build/reference/errorreport-report-internal-compiler-errors?view=msvc-170

Reply:
fwiw there is https://docs.microsoft.com/en-us/cpp/build/reference/diagnostics-compiler-diagnostic-options?view=msvc-170 which makes errors/warnings pretty-print in similar manner to clang (with `caret`)

Reply:
Interesting. 
It was one of the differences in the command line. cmake defaults it to queue, and visual studio defaults it to prompt. 

And I swear it made digging though backlog to find errors way easier.

Reply:
After restesting, it doesn't change anything, so I removed this.